sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql

```diff
@@ -1,436 +1,872 @@
-WITH overactive AS (
-  -- find client_ids with over 200,000 pings in a day
+/*
+This query depends on the following fields added to the main_v4 schema on 2019-11-22:
+
+  environment.addons.active_addons[].foreign_install
+  environment.addons.active_addons[].user_disabled
+  environment.addons.active_addons[].version
+  payload.simple_measurements.active_ticks
+  payload.simple_measurements.first_paint
+
+To backfill partitions for 2019-11-22 and earlier, use the last version of this query that read from main_summary_v4:
+https://github.com/mozilla/bigquery-etl/blob/813a485/sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql
+*/
+WITH base AS (
```

Comment:
> The final thing I want to see here before merging is clear documentation about the change from main_summary to main_v4 which specifies the first date for which the new logic is valid (and thus the limit for backfills using this methodology) and a link to the final version of the main_summary-based query so that it's easy to find should we need to do a longer backfill. This can probably be a comment at the top of the clients_daily query file.

how's this?